src/main/java/pl/sda/eventmanager/services/AdminPanelService.java

```diff
@@ -0,0 +1,24 @@
+package pl.sda.eventmanager.services;
+
+import org.springframework.stereotype.Service;
+import pl.sda.eventmanager.model.User;
+import pl.sda.eventmanager.repositories.UserRepository;
+
+import java.util.List;
+
+@Service
+public class AdminPanelService {
+
+    private final UserRepository userRepository;
+
+    public AdminPanelService(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    public List<User> findAll(){
+        List<User> allByOrderById = userRepository.findAllByOrderById();
+        allByOrderById.remove(0);
```

Comment:
1. I don't think you need to remove admin from user list.
2. To remove admin from list find him exclude him by role (using proper Spring Data method). Right now for someone who doesn't know logic of your app it's hard to understand why you're removing user with `id = 0`